### ISSUE TITLE
Exported in namespace

### ISSUE THREAD

**Issue by crackcomm** [dsherret/ts-morph#127]
I am struggling to filter out unexported `namespace`s and `functions` from within exported namespace.

It is pretty easily done from `SourceFile`:

```TypeScript
sourceFile.getFunctions()
      .filter(fn => fn.isNamedExport())

sourceFile.getNamespaces()
      .filter(ns => ns.isNamedExport())
      .map(ns => {
        // but here: ns.getNamespaces().filter(ns => ns.isNamedExport())
        // will be always empty
      })
```

Maybe you are able to help me somehow, thank you.

**Comment by crackcomm** [comment]
@dsherret thank you, `hasExportKeyword()` does exactly what I was looking for.

During development I tried to find `isExported()`, it may be a good idea to add it. It will replace use of all three functions in my case.

**Comment by dsherret** [comment]
Yes, I see how `isExported()` makes sense to add. I've opened #128.

**Comment by dsherret** [comment]
Just following up... `isExported()` is available in 0.92.0.

**Comment by crackcomm** [comment]
Thank you 👍
